cv32e20/tests/programs/custom/riscv_csr/gen_csr_test.py

```diff
@@ -0,0 +1,445 @@
+"""
```

Comment:
I won't gate the PR approval on this, but in future, anytime you make additions or modifications to an existing open-source file (with a permissive license), it is considered good practise to declare who owns the copyright on the modifications in the header.  A one liner comment on the top with your copyright declaration is sufficient.